### ISSUE TITLE
USA - State recognition doesn't compile

### ISSUE THREAD

**Issue by quincycs** [openalpr/openalpr#630]
Can someone provide a tip on how to compile & enable state reco?  Or point to a version / commit that it was working?

**Comment by kanakiyab** [comment]
I hope you were able to fix this. The headers are not imported for some _cout_ messages in two files. I am referencing your issue to mine #680, which has more details.

**Comment by quincycs** [comment]
Hi @kanakiyab , yeah I did the same and got it to compile.  However state recognition still didn't actually output.  I gave up after that and went with openALPR's commercial version.

**Comment by kanakiyab** [comment]
Thanks for your feedback, @quincycs . I am even unable to detect plates with the latest git pull. The alpr version is shown as `2.3.0` after installation, whereas the doc page says latest is `2.5`, did you have similar issue?

**Comment by quincycs** [comment]
Last time I installed from git pull it found plates properly but never states. I tried months ago though.  Today if I use the open source version, I install via apt-get.  

My guess is the documentation version is referring to the commercial version rather than the open source version.  The whole idea of their open source version also being called the same name as their commercial version is very confusing.  Good luck using “openALPR”
